src/misc.cpp

```diff
@@ -144,17 +143,41 @@ class Logger {
 /// Version is empty.
 
 string engine_info(bool to_uci) {
-
-  const string months("Jan Feb Mar Apr May Jun Jul Aug Sep Oct Nov Dec");
-  string month, day, year;
-  stringstream ss, date(__DATE__); // From compiler, format is "Sep 21 2008"
-
-  ss << "Stockfish " << Version << setfill('0');
-
-  if (Version.empty())
+  [[maybe_unused]] const string months("Jan Feb Mar Apr May Jun Jul Aug Sep Oct Nov Dec");
```

Comment:
It's [good practice to keep scopes small](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es5-keep-scopes-small), so maybe move to where it is needed (below the `#else`). Then you can also drop the `[[maybe_unused]]` attributes.

Reply:
This I can change yeah.